doc/example-config

```diff
@@ -1018,15 +1007,25 @@
 #
 # prepopulate-query-feeds yes
 
-####  ssl-verify
+####  ssl-verifyhost
+#
+# If no, skip SSL certificate host's name validation.
```

Comment:
I think it'd be better to describe what exactly "host's name validation" means. I think we can just copy and paste from [`CURLOPT_SSL_VERIFYHOST` docs](https://curl.haxx.se/libcurl/c/CURLOPT_SSL_VERIFYHOST.html):

> CURLOPT_SSL_VERIFYHOST - verify the certificate's name against host

(Don't forget to update configcommands.dsv.)